PHP/proxy.php

```diff
@@ -591,28 +719,27 @@ public function canProcessRequest()
         } else if ($this->proxyConfig['mustmatch'] == true || $this->proxyConfig['mustmatch'] == "true") {
 
             foreach ($this->serverUrls as $key => $value) {
+                
+                $serverUrl = $value['serverurl'][0];
 
-                $s = $value['serverurl'][0];
-
-                $s['url'] = $this->sanitizeUrl($s['url']); //Do all the URL cleanups and checks at once
-
-                if(is_string($s['matchAll'])){
-
-                    $mustmatch = strtolower($s['matchAll']);
+                $serverUrl['url'] = $this->sanitizeUrl($serverUrl['url']); //Do all the URL cleanups and checks at once
+                
+                if(is_string($serverUrl['matchall'])){
 
-                    $s['matchAll'] = $mustmatch;
+                    $serverUrl['matchAll'] = strtolower($serverUrl['matchall']);
```

Comment:
@phpmaps - should that not be lowercase matchall instead of matchAll with Capital A in ` $serverUrl['matchAll'] = strtolower...`